lib/core/src/sdk.rs

```diff
@@ -1621,12 +1616,12 @@ impl LiquidSdk {
                 bip353_address,
             } => {
                 let fees_sat = fees_sat.ok_or(PaymentError::InsufficientFunds)?;
-                let bolt12_invoice = self
+                let bolt12_info = self
                     .swapper
                     .get_bolt12_invoice(&offer.offer, *receiver_amount_sat)
```

Comment:
Nit: should we rename the method to e.g. `get_bolt12_info`? I'm also fine with keeping as is

Reply:
Fixed [2de8ff1](https://github.com/breez/breez-sdk-liquid/pull/907/commits/2de8ff19a09d1c8b36b93413b86638d8b2938f40)